forms-web-app/src/routes/check-answers.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const fetchExistingAppealMiddleware = require('../middleware/fetch-existing-appeal');
+
+const checkAnswersController = require('../controllers/check-answers');
+
+const router = express.Router();
+
+router.get('/', [fetchExistingAppealMiddleware], checkAnswersController.getCheckAnswers);
```

Comment:
an example of a route that uses the middleware to pre-populate the appeal form data